src/style.css

```diff
@@ -1 +1,95 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf) format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: bold;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+}
+
+.search-block {
+  margin: 20px 8px;
```

Comment:
The margin for `.search-block` is set, but it does not explicitly set spacing between search bars. Ensure that the spacing between search bars is explicitly defined as per the task requirements.